File: documentation/user/en/query/ordering/examples/natural/attribute-natural-non-localized.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	OrderBy(
        		AttributeNatural("orderedQuantity", Desc)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "orderedQuantity")
        		)
        	)
        )
	)
);

File: documentation/user/en/query/ordering/examples/comparable/attribute-natural-non-localized.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				orderBy(
					attributeNatural("orderedQuantity", DESC)
				),
				require(
					entityFetch(
						attributeContent("code", "orderedQuantity")
					)
				)
			)
		);
	}
);
